<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCollisionSimulationModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerCollisionSimulationModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="InputCollapsibleButton">
     <property name="text">
      <string>Input</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Input Mesh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="qMRMLNodeComboBox" name="inputMeshNodeComboBox">
          <property name="toolTip">
           <string>Mesh that will be dropped. The forces will be simulated on this mesh.</string>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="editEnabled">
           <bool>true</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
          <property name="selectNodeUponCreation">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inputMeshWarningLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Input mesh is NOT a volumetric mesh. Only volumetric mesh are supported.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input Floor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="floorMeshNodeComboBox">
        <property name="toolTip">
         <string>Mesh that will be used as the floor for the input mesh to collide with.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Output Mesh</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="outputMeshNodeComboBox">
        <property name="toolTip">
         <string>Model node that will receive the modified simulated data.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gravity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="Gravity">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="value">
         <double>9.800000000000001</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Contact Stiffness</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="Stiffness">
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Time Interval</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="Dt">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Youngs' Modulus</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Poisson Ratio</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="Youngs">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="Poisson">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>0.490000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Uniform Mass Value</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="Mass">
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="stopPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pausePushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
